Gems/ScriptCanvas/Code/Include/ScriptCanvas/Execution/ExecutionStateStorage.cpp

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <ScriptCanvas/Execution/ExecutionStateStorage.h>
+
+namespace ScriptCanvas
+{
+    void ExecutionStateStorage::Destroy()
+    {
+        Mod()->~ExecutionState();
```

Comment:
Rename to Destruct(). Add comment about the requirement to call Destroy() before the storage itself is deleted, but storage cannot do this, as it is not tracking the running status.